style.css

```diff
@@ -105,3 +105,413 @@ body {
   color: #f3f39c;
   transition-duration: 3ms;
 }
+
+.second-sec {
+  color: #e5e5e5;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  gap: 12px;
+}
+
+.second-tittle {
+  width: 327px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 700px;
+  font-size: 39px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #1c1a19;
+  margin-bottom: 30px;
+}
+
+.second-subtittle {
+  width: 327px;
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  display: flex;
+  align-items: center;
+  color: #091e42;
+}
+
+.second-p {
+  width: 327px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #091e42;
+}
+
+.list-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 0;
+  gap: 12px;
+  width: 297px;
+  height: 40px;
+}
+
+.css-btn {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  width: 49px;
+  height: 40px;
+  background-color: #fff;
+  border: 1px solid #8993a4;
+  font-family: 'Inter', serif;
```

Comment:
- [OPTIONAL] Please set up the font following the [Google fonts](https://fonts.google.com/) instruction using `<link>` or `@import`. Example: 
```
<link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500&family=Roboto:wght@100;300;400;500;700&display=swap" rel="stylesheet">
```
As not requested by the previous reviewer. I consider that optional. Check other fonts for the same issue.